android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -54,6 +58,19 @@ private static void throwEncodeException(JSONException e) {
     throw new RuntimeException("Could not encode JSON", e);
   }
 
+  public static String getApiKey(Context context){
+    int resourceId = context.getResources().getIdentifier("google_api_key", "string", context.getPackageName());
+    return resourceId != 0 ? context.getString(resourceId) : "";
+  }
+
+  public static String getProdPwsEndpoint(String key){
```

Comment:
These methods are a bit confusing in that it's weird that we would decide which endpoint to use based on what the key is.  Instead, let's make these all one method where you would pass a context and the pwcollection and set the values directly on the pwcollection.